documentation/service-rules/ecf1/void-declarations.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Lead provider API
+---
+
+This document outlines the process and implications of voiding participant declarations within the Early Career Framework (ECF) programme. It is intended for Lead providers interacting with the ECF API and internal DfE Finance team.
+
+This documentation describes:
+
+- What voiding means within the ECF service
+- Who can void a declaration and how
+- Validation rules and error messages
+- How voided declarations are displayed and reported
+
+## Contents
+
+[Definition of voiding](#definition-of-voiding)
+
+[Impact of voiding on declaration states](#impact-of-voiding-on-declaration-states)
+
+[Interactions](#interactions)
+
+[Lead provider voiding declarations](#lead-provider-voiding-declarations)
+
+[Error handling](#error-handling)
+
+[Lead provider viewing declarations](#lead-provider-viewing-declarations)
+
+[Finance users](#finance-users)
+
+[Frozen cohorts and statements](#frozen-cohorts-and-statements)
+
+## Definition of voiding
+
+**Voiding** is a mechanism that allows a Lead provider or a DfE Finance user to retract or cancel a previously submitted participant declaration.
+
+* This action is necessary in various scenarios, such as correcting errors, ineligible participants, responding to changes in participant circumstances, or when a declaration was made incorrectly.
+* A `voided` declaration is effectively cancelled and will not be used in invoicing or funding calculations.
+* Once a declaration is voided, it cannot be restored and must be re-submitted if appropriate.
+* A `voided` declaration will not be soft-deleted.
+
+## Impact of voiding on declaration states
+
+When a declaration is voided, its resulting state depends on the state it was in prior to the void action:
+
+* **Submitted, Eligible, Payable:** If a declaration is in a `submitted`, `eligible`, `payable` or `ineligible` state, voiding it will transition the declaration to a `voided` state with no further action required. It effectively nullifies the declaration. It will likely be marked as `voided` on any internal DfE listings or reports related to its original intended statement period, but it won't appear as a `payable` item.
+* **Paid:** If a declaration has already been `paid`, voiding it will transition the declaration to an `awaiting_clawback` state. This signifies that the funds paid out for this declaration need to be recouped by DfE, typically by being deducted from a future payment to the Lead provider. The clawback will be detailed on that future statement.
```

Comment:
Maybe it's worth explaining here how awaiting callback transitions to clawed back?